warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -45,6 +45,16 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
         """
         This will launch a browser.
 
+        If the user wants to test the selenium functionalities in headless
+        mode, install pyvirtualdisplay and Xvfb. To install it follow
+        the below steps.
+
+        1. To install pyvirtualdisplay:
+               pip install pyvirtualdisplay
+
+        2. To install Xvfb:
+               sudo apt-get install Xvfb
```

Comment:
Please also add a comment that shows how to install Xvfb on RedHat based systems (yum install)